aerogear/mobile-developer-console-operator
# feat(Monitor) : Add monitor for operator

## Motivation: 
- https://issues.jboss.org/browse/AEROGEAR-9115
- https://issues.jboss.org/browse/AEROGEAR-9117

## What
- Add monitor service for operator ( Prometheus + Grapahana )
- Add monitor make commands
- Add readme info. 

## Steps to verify
- Check: https://prometheus-route-middleware-monitoring.apps.london-2f09.openshiftworkshop.com/alerts

<img width="1630" alt="Screenshot 2019-07-23 at 10 44 44" src="https://user-images.githubusercontent.com/7708031/61717258-1d6d0000-ad37-11e9-8ea9-bb968d3d8016.png">

- Check: https://grafana-route-middleware-monitoring.apps.london-2f09.openshiftworkshop.com/d/null/mdc-operator?refresh=10s&orgId=1
